interface/patient_file/report/patient_report.php

```diff
@@ -218,6 +218,9 @@ function show_date_fun(){
    <input type='checkbox' name='include_transactions' id='include_transactions' value="transactions"><?php xl('Transactions','e'); ?><br>
    <input type='checkbox' name='include_batchcom' id='include_batchcom' value="batchcom"><?php xl('Communications','e'); ?><br>
   </td>
+     <td class="text">
+         <input type='checkbox' name='include_recurring_days' id='include_recurring_days' value="recurring_days" ><?php xl('Recurrent Appointments','e'); ?><br>
```

Comment:
use xlt(), use echo, and remove the e parameter:
<input type='checkbox' name='include_recurring_days' id='include_recurring_days' value="recurring_days" ><?php echo xlt('Recurrent Appointments'); ?><br>